src/platform/in-app-notification/in.app.notification.entity.ts

```diff
@@ -48,9 +57,202 @@ export class InAppNotification
   })
   receiverID!: string;
 
+  @ManyToOne(() => User, {
```

Comment:
Why have both the receiver and the receiverID? It should be one or the other I think.